integration/layer_reuse_test.go

```diff
@@ -75,33 +75,18 @@ func testLayerReuse(t *testing.T, context spec.G, it spec.S) {
 					settings.Buildpacks.MRI.Online,
 					settings.Buildpacks.Bundler.Online,
 					settings.Buildpacks.BundleInstall.Online,
-					settings.Buildpacks.BuildPlan.Online,
+					settings.Buildpacks.BundleList.Online,
 				)
 
-			firstImage, logs, err = build.Execute(name, source)
+			firstImage, _, err = build.Execute(name, source)
 			Expect(err).NotTo(HaveOccurred())
 
 			imageIDs[firstImage.ID] = struct{}{}
 
 			Expect(firstImage.Buildpacks).To(HaveLen(4))
 
 			Expect(firstImage.Buildpacks[2].Key).To(Equal(settings.Buildpack.ID))
-			Expect(firstImage.Buildpacks[2].Layers).To(HaveKey("gems"))
-
-			Expect(logs).To(ContainLines(
-				MatchRegexp(fmt.Sprintf(`%s \d+\.\d+\.\d+`, settings.Buildpack.Name)),
-				"  Executing build process",
-				"    Running 'bundle config --global clean true'",
-				MatchRegexp(fmt.Sprintf("    Running 'bundle config --global path /layers/%s/gems'", strings.ReplaceAll(settings.Buildpack.ID, "/", "_"))),
-				"    Running 'bundle config --global without development:test'",
-				"    Running 'bundle config --global cache_path --parseable'",
-				"    Running 'bundle install'",
-				MatchRegexp(`      Completed in \d+\.?\d*`),
-				"",
-				"  Configuring environment",
-				MatchRegexp(fmt.Sprintf(`    BUNDLE_USER_CONFIG -> "/layers/%s/gems/config"`, strings.ReplaceAll(settings.Buildpack.ID, "/", "_"))),
-				"",
-			))
+			Expect(firstImage.Buildpacks[2].Layers).To(HaveKey("launch-gems"))
```

Comment:
Should it also have a `build-gems` key at this point?

Reply:
It won't because the `build-gems` layer doesn't have `launch = true` set, so it won't appear in the launch image. This is part of the reason for the `bundle-list` buildpack. I needed a way to assert that that layer has the right gems.

Reply:
Oi gotcha